Stylesheet.css

```diff
@@ -1,41 +1,35 @@
-
-
-h1{
+h1 {
     text-align: center;
 }
 
-img{
+#images img {

```

Comment:
We want this to affect only specific images, and not all our images. If the page contains other images somewhere else, we don't want them to be affected.